speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/mainscreens/ResearcherMenu.cpp

```diff
@@ -21,10 +24,18 @@
 #define PRM_RECORD_BB_TGGLE  "CheckboxBBRecorderToggle"
 #define PRM_MAX_TIME         "MaxTimeEdit"
 #define PRM_USER_ID          "UserIdEdit"
+#define PRM_BLACKBOX         "ChooseBlackBoxButton"
 
 #define INDICATOR_AMOUNT 3
 #define PCONTROL_AMOUNT  6
 
+// Messages for file selection
+#define MSG_BLACK_BOX_NORMAL_TEXT   "Choose Black Box: "
+#define MSG_BLACK_BOX_NOT_EXE       "Choose Black Box: chosen file was not a .exe"
+#define MSG_BLACK_BOX_PATH_TOO_LONG "Choose Black Box: you've broken COM interface: your > 260 char file path was not aliased to an 8.3 file name"
```

Comment:
Maybe try to limit this error message to 'File too long (>260)' It runs off screen now
![image](https://user-images.githubusercontent.com/57985100/163683598-d8b44465-ffa1-40ea-8638-27a409c9134f.png)